contracts/solidity/upgrade.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# ANSI format
+GREEN='\e[32m'
+COLOR_RESET='\033[0m'
+
+cd "$(dirname "$0")"
+
+load_env() {
+    unamestr=$(uname)
+    if [ "$unamestr" = 'Linux' ]; then
+      export $(grep -v '^#' ./.env | xargs -d '\n')
+    elif [ "$unamestr" = 'FreeBSD' ] || [ "$unamestr" = 'Darwin' ]; then
+      export $(grep -v '^#' ./.env | xargs -0)
+    fi
+}
+load_env
```

Comment:
I would use the other technique of importing .env in a bash, as proposed here:
https://github.com/yetanotherco/yet-another-bridge/pull/64/files